administrator/components/com_admin/sql/updates/postgresql/5.2.2-2024-11-11.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "#__finder_links" ALTER COLUMN "route" TYPE VARCHAR(500);
```

Comment:
```suggestion
ALTER TABLE "#__finder_links" ALTER COLUMN "route" TYPE character varying(500);
```
In update SQL scripts we normally use `character varying` instead of `varchar` like in the installation SQL scripts. You can find examples in the 4.4-dev branch. Don't ask me why it is like that. I would suggest to stick with that.